src/v2i-hub/RSUHealthMonitorPlugin/src/RSUHealthMonitorWorker.cpp

```diff
@@ -145,7 +145,11 @@ namespace RSUHealthMonitor
                     }
                     else if (success)
                     {
-                        rsuStatuJson.append(populateJson(config.field, responseVal));
+                        auto json = populateJson(config.field, responseVal);
```

Comment:
What changed here for the RSU Health Monitor Plugin

Reply:
The JSON was an array. My intention is to create a JSON object.